Formula/ngrep.rb

```diff
@@ -15,16 +15,30 @@ class Ngrep < Formula
     sha256 cellar: :any_skip_relocation, el_capitan:    "d057c167d3b695ff915c13fd39e3cd7b3e6e2a5b3f82bce6bb8ea4c030e8f6e7"
   end
 
+  uses_from_macos "libpcap"
+
   def install
     sdk = MacOS.sdk_path_if_needed ? MacOS.sdk_path : ""
-    system "./configure", "--enable-ipv6",
-                          "--prefix=#{prefix}",
-                          # this line required to make configure succeed
-                          "--with-pcap-includes=#{sdk}/usr/include/pcap",
-                          # this line required to avoid segfaults
-                          # see https://github.com/jpr5/ngrep/commit/e29fc29
-                          # https://github.com/Homebrew/homebrew/issues/27171
-                          "--disable-pcap-restart"
+
+    args = ["--enable-ipv6",
+            "--prefix=#{prefix}",
+            # this line required to avoid segfaults
+            # see https://github.com/jpr5/ngrep/commit/e29fc29
+            # https://github.com/Homebrew/homebrew/issues/27171
+            "--disable-pcap-restart"]
```

Comment:
```suggestion
    args = [
      "--enable-ipv6",
      "--prefix=#{prefix}",
      # this line required to avoid segfaults
      # see https://github.com/jpr5/ngrep/commit/e29fc29
      # https://github.com/Homebrew/homebrew/issues/27171
      "--disable-pcap-restart",
    ]
```
Let's try to be a bit consistent with other formulae here.

Reply:
I should've flagged the trailing comma here 😅